lib/helpbase.gi

```diff
@@ -416,7 +416,7 @@ InstallGlobalFunction(SIMPLE_STRING, function(str)
 "efghijklmnopqrstuvwxyz[\000]^_\000abcdefghijklmnopqrstuvwxyz{ }~",
 "\177\200\201\202",
 "\203\204\205\206\207\210\211\212\213\214\215\216\217\220\221\222\223\224\225",
-"\226\227\230\231\232\233\234\235\236\237\238",
+"\226\227\230\231\232\233\234\235\236\237\240",
```

Comment:
\238 is not a valid octal escape, whereas \240 is (for the same number, if one were to allow overflows in the octal digits).